### PR TITLE
propose FIP 4 for remove pub address

### PR DESCRIPTION
**Pull request by edrotthoff**
this FIP proposes the new functionality of remove public address.

### CODE REVIEW

File: fip-4.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+## Specification
+### Remove FIO address
+#### New end point: *remove_fio_address* 
+#### New action in fio address contract remaddress
+##### Request
+|Parameter|Required|Format|Definition|
+|---|---|---|---|
+|fio_address|Yes|fio address, see FIO address validation rules.|FIO Address which will have public addresses removed.|
+|public_addresses|Yes|JSON Array. See public_addresses below. Min: 1 item Max: 5 item|The public address to be added to the FIO Address for the specified token.|
```
Comment (ericbutz) [inline fip-4.md:31]: The public addresses, and associated chain and token codes, to be removed.

Comment (pawmmm) [inline fip-4.md:31]: I don't think we should limit.

File: fip-4.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+```
+{
+	
+			"status": "Ok",
+			"fee_collected": "0"		
+}
+```
+## Fees
+a new fee will be created remove_pub_address, type 1, 600000000 SUF
+
```
Comment (pawmmm) [inline fip-4.md:93]: Is it bundle eligible? If so how many bundled transactions does it use?

File: fip-4.md (hunk 1 above)
Comment (edrotthoff) [reply fip-4.md:31]: I will correct this ...thanks

Comment (edrotthoff) [reply fip-4.md:31]: the limit will be removed..

File: fip-4.md (hunk 2 above)
Comment (edrotthoff) [reply fip-4.md:93]: I propose it use 1 bundled transaction. language will be added to the FIP to this effect.

**Pull request merged**